docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
       HASURA_GRAPHQL_ENABLED_APIS: "metadata,graphql"
       HASURA_GRAPHQL_ADMIN_SECRET: key
-      HASURA_GRAPHQL_JWT_SECRET: ${HASURA_GRAPHQL_JWT_SECRET}
+      HASURA_GRAPHQL_JWT_SECRET: "{\"type\": \"RS512\", \"key\": \"-----BEGIN CERTIFICATE-----\nMIIC7jCCAdagAwIBAgIJA8atJv4gkIatMA0GCSqGSIb3DQEBCwUAMB4xHDAaBgNV\nBAMTE3plbmlrYS5ldS5hdXRoMC5jb20wHhcNMTcwMzA3MDkzNTEzWhcNMzAxMTE0\nMDkzNTEzWjAeMRwwGgYDVQQDExN6ZW5pa2EuZXUuYXV0aDAuY29tMIIBIjANBgkq\nhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxax68QL7odBGpRzWEIo6MQzvSMGCepPj\nSYCMA17Azz6MZUMwfMdi7VD6V1dnP9mMbgGWaj6MFF5tAIRywM/Q5rc6ZuYvGBz/\nTZWfDfX19EarFO9O6czsM/Bj8AYJDINk2hON26qtPMihykndG8LkJgH60STzKkhF\n2xY7leVSqNVuAz1tWaRM2+7N4HLasaY3M69tK23ZMSkJXyfnxW31x8JtZNciSEXH\nS0gf27oAdYt9sqYbGfOom2x7lrirA4FWQKakbkb8yZTxyS9FiqgkVPAhH6+PbAx7\nD+TxOMTPDfRqxutjdZT5l0KdXmr4NWotiwk1cbr0Am8pN/JTHLQiBQIDAQABoy8w\nLTAMBgNVHRMEBTADAQH/MB0GA1UdDgQWBBShEaJbf9n8lF7UC5OtJ6PE1O3QqzAN\nBgkqhkiG9w0BAQsFAAOCAQEABIxvy9+tDgAtNfeJRKXd7QCI+FV2SPcoXwmpNimH\nxqPkYBJEoIY8tAQkGFSiDHYUpRhSnD1iacyrKxRlLzAgSRTm7eM80565HT87V/1I\nGAehPXla928y5P6bmZceT1KgbZas3YflnhAadm5ohlplK+d92+LTZ7530OPwwtCn\nHr7a/00v2vKdzCfY9VvdVU6Dhjuap969i3/JjhQAMvr796ArLIJb7zBfr859bWRc\n9BnBDQZhRlrUCb/rVJHOElymWdSnONntfitNOyXq+yl6NBYLM6tjlEBuvTX9L+bS\nP/+nOPRq0kOnUkc5fQ7PMRCZKE/54IYnXYO24aVObbNN5Q==\n-----END CERTIFICATE-----\n\"}"
   app:
```

Comment:
It's critical to leave the JWT secret here or not ?

Reply:
It's found easily only but I still think it has nothing to do here and should be in an env variable

Reply:
Why is this PR changing this?